### DISCUSSION TITLE
Oldguard!

### DISCUSSION THREAD

**Discussion by PatheticMustan** [Prodigy-Hacking/ProdigyMathGameHacking#860] [(D) Developer]
Hahahaha! You all thought I would do work, but it was me, burnedout!
Well anyways, apparently both Will and I are working on cheat menus, so uh

Let's release both and see which one fares better

hopefully oldguard will be more accessible, and will allow inexperienced developers to submit their own hacks without learning the shape of the boot

**Comment by MichealTheW12-4rd** [comment]
wait why do you support him then???

**Reply by ghost** [reply]
@PatheticMustan i hope yours is beter allso pls send me the link to both cheat menus

**Comment by ghost** [comment]
What does that mean

**Reply by MichealTheW12-4rd** [reply]
like if they re both doing the same thing maybe they might be rivals

**Comment by MichealTheW12-4rd** [comment]
Eh not interested

**Comment by NX721** [comment]
`Excited Lol`

**Comment by ghost** [comment]
I question everything
